pkg/eth/api_test.go

```diff
@@ -948,8 +958,17 @@ var _ = Describe("API", func() {
 			Expect(err).ToNot(HaveOccurred())
 			Expect(bal).To(Equal((*hexutil.Big)(common.Big0)))
 		})
-		It("Throws an error for an account it cannot find the balance for", func() {
-			_, err := api.GetBalance(ctx, randomAddr, rpc.BlockNumberOrHashWithHash(blockHash, true))
+		It("Retrieves the eth balance for the non-existing account address at the block with the provided hash", func() {
```

Comment:
How are we confirming expected behaviour (same question for all new assertions)?

Reply:
https://github.com/vulcanize/ipld-eth-server/pull/53#discussion_r645290501